src/planscape/forsys/merge_polygons.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.gis.geos import MultiPolygon, Polygon
+
+
+def merge_polygons(
```

Comment:
could we add a function description, particularly around the simplify part and its expected behavior?

Reply:
Done.